hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/checksum/ContainerChecksumTreeManager.java

```diff
@@ -92,28 +104,26 @@ public void markBlocksAsDeleted(KeyValueContainerData data, SortedSet<Long> dele
           .clearDeletedBlocks()
           .addAllDeletedBlocks(sortedDeletedBlockIDs)
           .build();
-      write(data, checksumInfoBuilder.build());
+      captureLatencyNs(metrics.getUpdateContainerMerkleTreeLatencyNS(),
+          () -> write(data, checksumInfoBuilder.build()));
```

Comment:
Can we move the read and write metrics inside the read and write methods so we don't need to duplicate this code every time they are called?